library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/optionselectormocks/MaterialCalendarGridView.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.google.android.material.datepicker
+
+import android.content.Context
+import android.view.ViewGroup
+
+internal class MaterialCalendarGridView(context: Context) : ViewGroup(context) {
```

Comment:
for `com.google.android.material.datepicker.MaterialCalendarGridView`, `com.google.android.material.timepicker.TimePickerView` and `androidx.appcompat.widget.DropDownListView ` you can add the necessary libraries directly to the test classpath using `testImplementation` dependency.

This will have an advantage of properly tracking name/package changes for these classes (instead of having our own versions).

Reply:
yes same as above you can't use them as they are either not public or `hidden`

Reply:
Here I'm not talking about using them directly, we are referencing them by name anyway. Here I mean that we don't need to create mocks, we can have them in the classpath by adding the libraries to the test classpath only.